### PR TITLE
Use longitude-one/wkt-parser, remove php 7.4 & 8.0 support

### PR DESCRIPTION
**Pull request by axi**
- min php version >= 8.1 (as in longitude-one/wkt-parser)
- use Attributes for tests as doctrine/annotation 2 removed SimpleAnnotationReader
- on my linux, I had to change some on the tests values to make it pass, not included in this pull request because I found a comment "//TODO: Test should be fixed, distance are differents on Windows and on Linux." 

There is probably more to do but that can be a start

### CODE REVIEW

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -1,24 +1,37 @@
 <?php
+/**
+ * This file is part of the doctrine spatial extension.
+ *
+ * PHP 7.4 | 8.0 | 8.1
```
Comment (ellisgl) [inline .php-cs-fixer.dist.php:5]: Thought we wanted to do 8.1+?

**Review by ellisgl — APPROVED** [review]
Look good to me.

**Comment by axi** [comment]
tests should be ok now, I had to modify workflow so forks can be executed (so it need maintener approval to be run)

**Review by ellisgl — APPROVED** [review]
Looks like pretty much auto updated ode and then a couple fixes around tests that look to have passed.

**Comment by ca-mk** [comment]
+1

**Comment by pimhakkert** [comment]
Hey guys, I just stumbled upon this library, would love to use it, but obviously can't because I'm running the latest doctrine version and PHP 8.1.

When do you think this will be merged into the main branch? Thanks!

**Comment by misaert** [comment]
> Hey guys, I just stumbled upon this library, would love to use it, but obviously can't because I'm running the latest doctrine version and PHP 8.1.
> 
> When do you think this will be merged into the main branch? Thanks!

Hi Pim,

In our side, we used this library with PHP8.1 and Doctrine ORM 2.14.2 without worries. 

It just Doctrine Annotations and Doctrine Lexer which are in the previous versions (see #36).

What is your issue?

Regards,

Mickaël

**Pull request merged**